src/index.js

```diff
@@ -45,3 +45,22 @@ exports.Queue = Plotly.Queue;
 
 // export d3 used in the bundle
 exports.d3 = require('d3');
+
+Plotly.register({
+    traces: {
+        bar: require('./traces/bar'),
```

Comment:
it would be nice to make it clear what is registerable though. 

If something is registerable then how does Plotly register it? If there is a difference between components and traces perhaps two functions is the way to go. IF there isn't much of a difference or if the difference doesn't matter than a single function should do.